network/test/cohort1/network_test.go

```diff
@@ -199,7 +199,7 @@ func (suite *NetworkTestSuite) TestUpdateNodeAddresses() {
 
 	// create a new staked identity
 	ids, libP2PNodes := testutils.LibP2PNodeForNetworkFixture(suite.T(), suite.sporkId, 1)
-	idProvider := unittest.NewUpdatableIDProvider(ids)
+	idProvider := unittest.NewUpdatableIDProvider(append(suite.ids, ids...))
```

Comment:
I have no idea how this test originally worked since it was completely broken. maybe the networking logic was so slow that the message was never actually delivered. 🤷